.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for review when someone
+# opens a pull request.
+* @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
Do we want to trim this list of code owners down a bit?

Reply:
I think we are all familiar with this one...at least we all will be after @hillaryj's PR is merged.